AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m

```diff
@@ -2634,24 +2634,20 @@ - (void)testOpenURLInAuthenticationSessionFails API_AVAILABLE(ios(11)) {
 }
 
 - (void)testDependencyCallUsesInjectedHttpClient {
-    id httpClientMock = [MSHttpClient new];
-    [MSDependencyConfiguration setHttpClient:httpClientMock];
-    
-    // If
-    self.sut.appSecret = kMSTestAppSecret;
-    id distributeMock = OCMPartialMock(self.sut);
-    OCMReject([distributeMock handleUpdate:OCMOCK_ANY]);
-    id reachabilityMock = OCMClassMock([MS_Reachability class]);
-    OCMStub([reachabilityMock reachabilityForInternetConnection]).andReturn(reachabilityMock);
-    OCMStub([reachabilityMock currentReachabilityStatus]).andReturn(ReachableViaWiFi);
-   
-    // When
-    [distributeMock checkLatestRelease:kMSTestUpdateToken distributionGroupId:kMSTestDistributionGroupId releaseHash:kMSTestReleaseHash];
-    
-    // Then
-    OCMReject([MSHttpClient new]);
-    [reachabilityMock stopMocking];
-    [distributeMock stopMocking];
+
+  // If
+  id httpClient = OCMClassMock([MSHttpClient class]);
+  [MSDependencyConfiguration setHttpClient:httpClient];
+  self.sut.appSecret = kMSTestAppSecret;
+  id distributeMock = OCMPartialMock(self.sut);
+
+  // When
+  [distributeMock checkLatestRelease:kMSTestUpdateToken distributionGroupId:kMSTestDistributionGroupId releaseHash:kMSTestReleaseHash];
```

Comment:
In this test we just verify we set our own http client then get it internally.
Thus this line achieves nothing. Either also verify interaction on http client or remove this line.

Reply:
The unit test wants to verify that it is using injected HTTP client. Actual interactions would be verified by other unit tests that we already have.